Arithmetic.pp

```diff
@@ -65,7 +66,10 @@
     ternary() ( ::times:: #multiplication expression() )?
 
 ternary:
-    term() ( ::div:: #division expression() )?
+    quaternary() ( ::div:: #division expression() )?
+
+quaternary:
+    term() ( ::pow:: #power expression() )?
```

Comment:
As said [here](http://php.net/manual/en/migration56.new-features.php#114892),
* the common `^` operator is left associative (according to my mind and my calculator (2^3^2=64))
* the PHP `**` operator is right associative (2 ** 3 ** 2 == 512).

What about `Hoa\Math` `^` operator ?